source/_integrations/autarco.markdown

```diff
@@ -33,3 +33,10 @@ Gain insight into how much solar energy you produce per site linked to your acco
 - Energy production today (kWh)
 - Energy production this month (kWh)
 - Energy production total (kWh)
+
+### Inverters
+
+Gain insight into how much energy each inverter produces, that is linked to a site.
```

Comment:
what do you mean by "that is linked to a site"?

Reply:
An account you add can have multiple "sites" and each site has 1 or more inverters linked to it. For example, each site could be a separate house, but Autarco itself uses the term "sites" on their platform.